hw/ip/otbn/doc/_index.md

```diff
@@ -921,6 +921,7 @@ The host CPU can determine if an application has completed by either polling {{<
 * To poll for a completed operation, software should repeatedly read the {{< regref "STATUS" >}} register.
   While the operation is in progress, {{< regref "STATUS" >}} is non-zero.
   The operation is complete if {{< regref "STATUS" >}} is `IDLE`.
+  If a fatal error occurs {{< regref "STATUS" >}} will become `LOCKED` and the operation will never complete.
```

Comment:
We have used term "the operation completes" to indicate that the `done` interrupt is sent, and the status changes away from busy to either idle or (now new:) locked. In the case of a fatal error during an operation, it is meant to complete, but leave OTBN locked and not idle.